src/EditorFeatures/Test2/NavigableSymbols/NavigableSymbolsTest.vb

```diff
@@ -45,6 +46,32 @@ class {|target:C|}
             End Using
         End Function
 
+        <WpfFact, Trait(Traits.Feature, Traits.Features.NavigableSymbols)>
+        Public Async Function TestCharpLiteral() As Task
+            Dim markup = "var x = {|literal:1$$23|};"
```

Comment:
📝 It seems like the `{|literal:1$$23|}` could be simplified to just `1$$23`. I'm not sure the designation as literal helps anything.